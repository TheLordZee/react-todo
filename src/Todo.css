.Todo {
    border: 2px solid black;
    padding: 3vh;
}

.Todo-text {
    display: inline;
}

.Todo-btn {
    margin-left: 10vw;
}
