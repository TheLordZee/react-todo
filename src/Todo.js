import React from "react"
import "./Todo.css"

const Todo = ({id, text, removeTodo}) => (
    <div id={id} className="Todo">
        <p className="Todo-text">{text}</p>
        <button className="Todo-btn" onClick={removeTodo}>X</button>
    </div>
) 

export default Todo;