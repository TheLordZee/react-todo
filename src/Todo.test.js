import { render } from '@testing-library/react';
import Todo from "./Todo"

it('renders', () => {
  render(<Todo text="Test todo" />);
});

it("matches snapshot", () => {
    const {asFragment} = render(<Todo text="Test todo" />);
    expect(asFragment()).toMatchSnapshot();
})