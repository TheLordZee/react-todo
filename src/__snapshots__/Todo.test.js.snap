// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
    <p
      class="Todo-text"
    >
      Test todo
    </p>
    <button
      class="Todo-btn"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
